app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailPresenter.java

```diff
@@ -55,7 +55,13 @@ public void loadDetails(boolean forceReload) {
 
         chartAnalyser.loadData(getId())
             .compose(disposeCompletable(getDisposable()))
-            .subscribe(() -> chartAnalyser.showChart(getView().getChartView()), Logger::logError);
+            .subscribe(() -> {
+                getView().showChart(true);
+                chartAnalyser.showChart(getView().getChartView());
+            }, (throwable) -> {
+                Logger.logError(throwable);
```

Comment:
Throwable can be put out of the brackets. And Logger is only used when we don't provide any action on error. It's just a proxy